scripts/run-dependencies.sh

```diff
@@ -12,7 +12,7 @@ HINTR_VERSION=$(<$HERE/../src/config/hintr_version)
 
 REGISTRY=mrcide
 DB_IMAGE=$REGISTRY/hint-db:master
-DB_MIGRATE_IMAGE=$REGISTRY/hint-db-migrate:master
+DB_MIGRATE_IMAGE=$REGISTRY/hint-db-migrate:mrc-1757
```

Comment:
Will this need to be changed back to master when you merge?

Reply:
Yes, will do that after merge.

Reply:
Oh actually, that db branch is already merged, I'll make that change here.